datalad/crawler/main.py

```diff
@@ -232,7 +233,8 @@ def page2annex(self, existing='check', dry_run=False, cache=False):
                         incoming_filename, incoming_downloaded, incoming_updated, downloaded_size = \
                           download_url_to_incoming(href_full, incoming_annex.path,
                                        join(repo_sectiondir, href_dir),
-                                       db_incoming=db_incoming, dry_run=self.runner.dry,  # TODO -- use runner?
+                                       db_incoming=db_incoming,
+                                       dry_run=self.runner.protocol.__class__.__name__ == 'DryRunProtocol',  # TODO -- use runner?
```

Comment:
yikes... ```isinstance(self.runner.protocol, DryRunProtocol)``` 

but in the long run even that one wouldn't work whenever we come up with "CompositeProtocol``` or alike to join use multiple protocols at the same time.... just something to keep in mind -- can't be addressed ATM and probably this would be RFed anyways